contributors/guide/pull-requests.md

```diff
@@ -109,6 +110,24 @@ easily see all criteria that are not being met and address them.
 
 That's the last step. Your pull request is now merged.
 
+## More About `Ok-To-Test`
+
+- For a Contributor, A `ok-to-test` label may just mean regular CI Testing to be done.
+- For the reviewer or the member, labelling the PR with `ok-to-test` it means a lot more:
+    + They need to take care if the PR is not a wastage of our `CI/CD` resources.
+    + Is the PR worth testing or does it needs more change before going through the `CI/CD` process ?
+    + Is the PR getting used to run malicious code to misuse our resources ?
+- An `ok-to-test` label may reduce the workload and smoothens the contributors experience as they can know if there is any failing test. If there is, you can fix the test and they don't have to wait for a long time to get a review from `maintainer/assignee`.
+- There are various other factors on which labelling of `ok-to-test` depends :
+    + Size of PR :
+        * If the PR is of `size/S` or `size/M` which is just to fix a flake or test or spelling mistake, the reviewer can trigger the `CI/CD` without having a second thought.
```

Comment:
I'm hesitant on this one, it's usually okay for things like spelling, grammer, or link fixes -- what we refer as [trivial edits](https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#10-trivial-edits).

Tests, flakes etc should still be evaluated.